### PR TITLE
2382: Hide expired events

### PR DESCRIPTION
**Pull request by SteffiStuffel**
### Resolved issues

Fixes: #2382 

---

### CODE REVIEW

File: api-client/src/endpoints/createEventsEndpoint.ts [hunk 1]
```diff
@@ -125,7 +132,9 @@ const createRecurringEvents = (event: JsonEventType): JsonEventType[] => {
     })),
     event: {
       ...event.event,
+      start: date.toISOString(),
```
Comment (LeandraH) [inline api-client/src/endpoints/createEventsEndpoint.ts:135]: Do we really need `start` and `start_date` both?

File: api-client/src/endpoints/createEventsEndpoint.ts [hunk 2]
```diff
@@ -125,7 +132,9 @@ const createRecurringEvents = (event: JsonEventType): JsonEventType[] => {
...
       start_date: dateToString(date),
+      end: getEndDate(event, date).toISOString(),
```
Comment (LeandraH) [inline api-client/src/endpoints/createEventsEndpoint.ts:137]: Same for `end` and `end_date`?

**Review by LeandraH — APPROVED** [review]
Nice, tested in Firefox

File: api-client/src/endpoints/createEventsEndpoint.ts (hunk 1 above)
Comment (SteffiStuffel) [reply api-client/src/endpoints/createEventsEndpoint.ts:135]: We do not really "need" both, but as this function updates these values for recurring events, i think it makes sense and the code more robust if both values are set correctly.

Comment (LeandraH) [reply api-client/src/endpoints/createEventsEndpoint.ts:135]: Maybe add a comment explaining that? I definitely see myself at some point attempting to remove one of these 😅

Comment (f1sh1918) [reply api-client/src/endpoints/createEventsEndpoint.ts:135]: Sorry also to ask. I didn't get the purpose yet for it. Where do we want to use `start_date`. Even these attributes are not listed in `JsonEventType[]`

Comment (steffenkleinle) [reply api-client/src/endpoints/createEventsEndpoint.ts:135]: Same, we shouldn't need both :see_no_evil:

File: api-client/src/endpoints/createEventsEndpoint.ts [hunk 3]
```diff
@@ -125,10 +132,11 @@ const createRecurringEvents = (event: JsonEventType): JsonEventType[] => {
     })),
     event: {
       ...event.event,
-      start_date: dateToString(date),
-      end_date: dateToString(getEndDate(event, date)),
+      start: date.toISOString(),
+      end: getEndDate(event, date).toISOString(),
     },
   }))
+  //console.log(events[0].event)
```
Comment (steffenkleinle) [inline api-client/src/endpoints/createEventsEndpoint.ts:139]:
```suggestion
```

**Review by steffenkleinle — COMMENTED** [review]
Tested on firefox

**Pull request merged**